src/library_webgl2.js

```diff
@@ -723,6 +723,22 @@ var LibraryWebGL2 = {
     GL.syncs[id] = null;
   },
 
+#if WASM_BIGINT
+  glClientWaitSync__sig: 'iiii',
+  glClientWaitSync: function(sync, flags, timeout) {
```

Comment:
I think this kind of thing can be handled without duplicating the functions, by using `{{{ defineI64Param('timeout') }}}` and related functions to read from the parameter or parameter pair.

Reply:
(grep for `defineI64Param` for examples)

Reply:
Updated to avoid function duplication. Unfortunately the existing functions for parameter pairs are not fully adequate for this purpose, though `{{{ defineI64Param('timeout') }}}` works well.